statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/RightsCategory.java

```diff
@@ -0,0 +1,52 @@
+package eu.europeana.statistics.dashboard.common.iternal;
+
```

Comment:
package name iternal? i think you intended ```internal```

Reply:
Uh oh, this needs to be fixed :p

Reply:
remember to add unit test to this new class.

Reply:
Added